api/lib/api/clients/common.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Tesla.Middleware.RequestLogger do
+  @behaviour Tesla.Middleware
+
+  def call(env, next, _) do
+    env
+    |> IO.inspect()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [There should be no calls to IO.inspect/1.](https://app.codacy.com/manual/lucaskjaero/foreign-language-reader/pullRequest?prid=5112709)